org.scala-ide.sdt.core/src/org/scalaide/ui/editor/DecoratedInteractiveEditor.scala

```diff
@@ -35,15 +48,19 @@ trait DecoratedInteractiveEditor extends ISourceViewerEditor {
     // http://stackoverflow.com/questions/12507620/race-conditions-in-annotationmodel-error-annotations-lost-in-reconciler
     val presViewer = getViewer
```

Comment:
do we need this line? `presViewer` is actually not used. This is not the must of course.